apps/ensrainbow/src/index.test.ts

```diff
@@ -84,24 +84,14 @@ describe("ENS Rainbow API", () => {
 
     it("should return correct count of entries", async () => {
       // Add some test data
-      const testData = [
-        {
-          hash: "0x1111111111111111111111111111111111111111111111111111111111111111",
-          label: "test1",
-        },
-        {
-          hash: "0x2222222222222222222222222222222222222222222222222222222222222222",
-          label: "test2",
-        },
-        {
-          hash: "0x3333333333333333333333333333333333333333333333333333333333333333",
-          label: "test3",
-        },
-      ];
+      const testData = ["test1", "test2", "test3"].map(label => ({
+        hash: labelhash(label),
```

Comment:
Nice improvements here! Please see my comment above for more details.

```suggestion
        labelHashBytes: labelHashBytes(label),
```